qemu.nsi

```diff
@@ -216,9 +257,9 @@ Section "Uninstall"
     DeleteRegKey HKLM SOFTWARE\${PRODUCT}
 
     ; Remove shortcuts, if any
-    Delete "$SMPROGRAMS\${PRODUCT}\User Documentation.lnk"
-    Delete "$SMPROGRAMS\${PRODUCT}\Technical Documentation.lnk"
-    Delete "$SMPROGRAMS\${PRODUCT}\Uninstall.lnk"
+    Delete "$SMPROGRAMS\${PRODUCT}\$(User_Documentation_Link_Text).lnk"
```

Comment:
The names of these registry keys are defined by Microsoft. They must not be translated.

Reply:
It seems not registry about program functions but program links (like icon description for program group).
Then if it's true you can localize them because are icon descriptions in program group.